src/platform/datapath_winuser.c

```diff
@@ -1048,6 +1053,105 @@ CxPlatDataPathIsPaddingPreferred(
     return !!(Datapath->Features & CXPLAT_DATAPATH_FEATURE_SEND_SEGMENTATION);
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+_Success_(QUIC_SUCCEEDED(return))
+QUIC_STATUS
+CxPlatDataPathGetGatewayAddresses(
```

Comment:
Should this have some sort of SAL annotation that it allocates GatewayAddresses?